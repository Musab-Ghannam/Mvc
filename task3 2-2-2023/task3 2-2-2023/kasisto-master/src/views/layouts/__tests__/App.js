/* eslint-env jest */

import React from 'react'
import AppLayout from '../'

import { Provider } from 'react-redux'
import renderer from 'react-test-renderer'
import configureMockStore from 'redux-mock-store'
import thunk from 'redux-thunk'

const middlewares = [thunk]
const mockStore = configureMockStore(middlewares)

it('experiments with Jest', () => {
  const store = mockStore()
  const component = renderer.create(
    // {... { actions, payment }}
    <Provider store={store}>
      <AppLayout />
    </Provider>
  )
  let tree = component.toJSON()

  // expect(Payment.on).toBeCalledWith('ready')

  expect(tree).toMatchSnapshot()
})
