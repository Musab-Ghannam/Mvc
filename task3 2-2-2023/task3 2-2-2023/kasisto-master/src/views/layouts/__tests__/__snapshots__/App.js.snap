// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`experiments with Jest 1`] = `
<div
  className="o-app"
/>
`;
